CONTRIBUTING.md

```diff
@@ -67,9 +67,15 @@ complete correctly with the unpatched version, to make sure the test case is cov
 
 ### Performance
 
+Any change which could have an impact on Alacritty's performance, should be tested to prevent
+potential regressions.
+
 Alacritty mainly uses the [vtebench](https://github.com/jwilm/vtebench) tool for testing Alacritty's
-performance. Any change which could have an impact on Alacritty's performance, should be tested with
-it to prevent potential regressions.
+performance. Instructions on how to use it can be found in its
+[README](https://github.com/jwilm/vtebench/blob/master/README.md).
+
```

Comment:
Could we add a point to LLVM benchmarking on how to setup systems?

Reply:
Well, having this info on vtebench side probably makes more sense... Let's just skip it for now, unless folks will be constantly ignoring this part of disabling CPU frequency scaling things.

Reply:
Yeah, we can only hope that people read documentation anyways, there's no guarantee no matter where we put it.

And they need to read the vtebench readme anyways, to figure out how to use it.